CHANGELOG.md

```diff
@@ -7,8 +7,7 @@ This document contains a historical list of changes between releases. Only
 changes that impact end-user behavior are listed; changes to documentation or
 internal API changes are not present.
 
-Main (unreleased)
------------------
+## Main (unreleased)
```

Comment:
@Caleb-Hurshman just noting that all these formatting changes might be overstepping for this PR. i don't think it's a huge deal though and you can see what they say

Reply:
Agreed, let's please leave the changelog as-is and just add the new entry.

Reply:
A classic format-on-save mishap